src/coordinator/roles.js

```diff
@@ -46,6 +46,102 @@ class Roles {
     this._sdk = sdk;
   }
 
+  /**
+   * Add an application to a role
+   *
+   * API Endpoint: '/applications/:applications_id/roles/:roleId'
+   * Method: POST
+   *
+   * @param {string} organizationId The ID of the organization
```

Comment:
Are you just being proactive with asking for this organizationId without actually using it yet?

Reply:
yes